www/src/css/snaplist.css

```diff
@@ -15,6 +15,16 @@
   margin-right:10px;
 }
 
+.b-store .b-installer {
```

Comment:
This is the same issue as https://github.com/snapcore/snapweb/pull/21#discussion_r70393955 and is related to the use of Handlebar's partial templates.

I've just read through the documentation and it seems [it's possible to pass parameters to the partial](http://handlebarsjs.com/partials.html#partial-parameters). I agree that it would be ideal to remove the use of CSS class nesting so I'll see if I can rework this and the other PR to pass a modifier class to use in the partial.